src/js/components/List/stories/children.js

```diff
@@ -20,7 +25,9 @@ export const ChildrenExample = () => (
             align="center"
           >
             <Gremlin size="large" />
-            <Text weight="bold">{datum}</Text>
+            <Tip content={datum.state} dropProps={{ align: { left: 'right' } }}>
```

Comment:
Can we put the Tip containing the Box, at line 20?

I'm also thinking that this story doesn't feel very "real". What about shrinking the List to feel more like it's inside a small panel, such that there isn't room for more than the icon + name. As it stand, the List is so large that there's plenty of room for the city to be shown. The Tip content feels a bit underwhelming currently.

Reply:
Fixed in #4705.

Changed the panel to be smaller, so Tooltip will make more sense.